training/dataset.py

```diff
@@ -358,10 +359,13 @@ def _find_nearest_resolution(self, height, width):
 
 
 class BucketSampler(Sampler):
-    def __init__(self, data_source: VideoDataset, batch_size: int = 8, shuffle: bool = True) -> None:
+    def __init__(
+        self, data_source: VideoDataset, batch_size: int = 8, shuffle: bool = True, drop_last: bool = False
+    ) -> None:
         self.data_source = data_source
         self.batch_size = batch_size
         self.shuffle = shuffle
+        self.drop_last = drop_last
```

Comment:
How significant is this?

Reply:
I think good to have in case one does not want to use incomplete buckets that don't yet have `args.train_batch_size` number of entries

Reply:
Then let's make a note about it in a comment.

Reply:
completed